homeassistant/components/conversation/__init__.py

```diff
@@ -39,6 +39,21 @@
     extra=vol.ALLOW_EXTRA,
 )
 
+WS_TYPE_GET_ONBOARDING = "conversation/onboarding/get"
```

Comment:
Could we embed onboarding info into the attribution and call that command `conversation/agent/info` ?